migrations/20230202165836-create-user.js

```diff
@@ -9,11 +9,11 @@ module.exports = {
         primaryKey: true,
         type: DataTypes.INTEGER
       },
-      first_Name: {
+      first_name: {
         type: DataTypes.STRING,
         allowNull: false,
       },
-      last_Name: {
```

Comment:
Are these updates autogenerated from the schema change locally or did you manually change this?

Reply:
I manually changed this, because this incorrect format of variable name